tests/qos/qos_sai_base.py

```diff
@@ -21,7 +21,7 @@ class QosBase:
     SUPPORTED_T0_TOPOS = ["t0", "t0-64", "t0-116", "t0-35", "dualtor-56", "dualtor", "t0-80", "t0-backend"]
     SUPPORTED_T1_TOPOS = ["t1-lag", "t1-64-lag", "t1-backend"]
     SUPPORTED_PTF_TOPOS = ['ptf32', 'ptf64']
-    SUPPORTED_ASIC_LIST = ["gb", "td2", "th", "th2", "spc1", "spc2", "spc3", "td3", "th3"]
+    SUPPORTED_ASIC_LIST = ["gb", "td2", "th", "th2", "spc1", "spc2", "spc3", "td3", "th3", "dnx"]
```

Comment:
![image](https://user-images.githubusercontent.com/46945843/188723502-e5307898-81b1-4e8e-814e-d7e718ee9397.png)

Reply:
Shouldn't we add asic type as "jr2" or "j2c+" ? Also we may need to add "SUPPORTED_T2_TOPOS.

Reply:
added j2c+ to supported ASIC list